.vscode/extensions.json

```diff
@@ -1,3 +1,3 @@
 {
-  "recommendations": ["google.wireit"]
+  "recommendations": ["google.wireit", "GitHub.vscode-github-actions"]
```

Comment:
This extension is context aware so if you input the wrong SHA it will show error.